controllers/authenticationController.js

```diff
@@ -0,0 +1,95 @@
+const { Op } = require("sequelize");
+const db = require("../models/index");
+require("dotenv").config();
+const {encrypt}= require('../utils/encryption')
+const Customer = require("../models").customer;
+let jwt = require("jsonwebtoken");
+
+
+
+const createUser = (req, res) => {
+
+    console.log(req.body)
+
+    
+    let { userName, password } = JSON.parse(JSON.stringify(req.body));
+  
+    const userNameEncrypted = encrypt(userName);
+    const passwordEncrypted = encrypt(password);
+  
+    return db.customer
+      .create({
+        name: userNameEncrypted,
+        password: passwordEncrypted,
```

Comment:
@osama891 Use es6 de-structuring instead of repetition of req.body......

Reply:
Refactoring is required. I'll do it today